redux/auth/actions.ts

```diff
@@ -35,3 +43,39 @@ export function logout(): LogoutAction {
     type: LOGOUT,
   };
 }
+
+export function signup(
+  email: string,
+  password: string,
+  phone?: string
+): SignupAction {
+  return {
+    type: SIGNUP,
+    payload: { email, phone, password },
+  };
+}
+
+export function signupSuccess(token: string): SignupSuccessAction {
+  return {
+    type: SIGNUP_SUCCESS,
+    payload: { token },
+  };
+}
+
+export function signupFailure(message: string): SignupFailureAction {
+  return {
+    type: SIGNUP_FAILURE,
+    payload: message,
+  };
+}
+
+export function setUserData(data): SetUserDataAction {
```

Comment:
Please remove this action creator and related code as stated in the comment in `redux/auth/saga.ts` file.